src/rdydm.c

```diff
@@ -157,6 +164,9 @@ PetscErrorCode CreateDM(RDy rdy) {
 
   PetscCall(DMViewFromOptions(rdy->dm, NULL, "-dm_view"));
 
+  // rename the fields in the distributed section
```

Comment:
@knepley , here's where we rename the fields after the distribution. We can remove this when the bug you mentioned is fixed.